tools/clang/test/CodeGenHLSL/RValSubscript.hlsl

```diff
@@ -1,5 +1,7 @@
 // RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
 
+// CHECK: alloca [16 x i32]
```

Comment:
I believe the old implementation created an alloca not at the alloca insertion point, hence the ordering change.